4-spring-web-app/1-Authentication/sign-in/README.md

```diff
@@ -82,9 +82,9 @@ This sample demonstrates a Java Spring MVC web app that signs in users on your M
 From your shell or command line:
 
 ```console
```

Comment:
Any updates on the readme shouldn't be needed ... as in, these updates would need to happen in the docs. README.md here should be completely emptied and should link back to the relevant doc article. 
Although it might be good to have the needed updates here as well, so we know what we need to update on the docs article ...

Reply:
Yes, this readme update is nice to have